text_collector_examples/smartmon.sh

```diff
@@ -7,6 +7,10 @@
 #       data in them than you'd think.
 #       http://arstechnica.com/civis/viewtopic.php?p=22062211
 
+PATH="/usr/sbin:$PATH"
```

Comment:
Let's leave this out, it's up to the user to make sure they give the script a valid PATH.

Reply:
Deleted